pkg/controllers/dnspolicy/dnspolicy_controller.go

```diff
@@ -44,10 +44,12 @@ import (
 )
 
 const (
-	DNSPolicyFinalizer                                    = "kuadrant.io/dns-policy"
-	DNSPoliciesBackRefAnnotation                          = "kuadrant.io/dnspolicies"
-	DNSPolicyBackRefAnnotation                            = "kuadrant.io/dnspolicy"
-	DNSPolicyAffected            conditions.ConditionType = "kuadrant.io/DNSPolicyAffected"
+	DNSPolicyFinalizer                                           = "kuadrant.io/dns-policy"
+	DNSPoliciesBackRefAnnotation                                 = "kuadrant.io/dnspolicies"
+	DNSPolicyBackRefAnnotation                                   = "kuadrant.io/dnspolicy"
+	DNSPolicyAffected                   conditions.ConditionType = "kuadrant.io/DNSPolicyAffected"
+	DNSPolicyHealthChecksNoIgnoredHosts                          = "NoIgnoredHosts"
+	DNSPolicyHealthChecksIgnoredReason                           = "NoGatewaysWithWildcardHosts"
```

Comment:
I am finding the reason here hard to reason about. Should it not be `DNSPolicyHealthChecksIgnoredReason = GatewayHasWildcardHosts`?

Reply:
yeah, this is a bit confusing, I can't recall why I phrased it like this... It looks mental. I'll update it to be more readable.